GMnetENGINE.gmx/scripts/htme_clientStart.gml

```diff
@@ -37,6 +37,31 @@ if( self.socketOrServer<0 ){
     return false;
 }
 
+// Get server port from list

```

Comment:
Yes. I tried to not to change the demo objects only the scripts so thats why this seems not right. I dont know why I am afraid of changing the demo objects :-)